debian/postinst

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+case "${1}" in
+    triggered)
+        for trigger in ${2}; do
+            case "${trigger}" in
+
+                /etc/xdg/xfce4/xinitrc)
+                    debug "Setting XFCE Qubes App Menus..."
+                    sed -i 's|XDG_MENU_PREFIX="xfce-"|XDG_MENU_PREFIX="xfce-qubes-"|g' /etc/xdg/xfce4/xinitrc
```

Comment:
Looking at that xinitrc script, it sets XDG_MENU_PREFIX only if it isn't set already. This gives a hope for setting it somewhere earlier without modifying a file from another package. Perhaps `/etc/X11/xinit/xinitrc.d`?

Reply:
Yes like we do in the case of XFCE template: https://github.com/fepitre/qubes-gui-agent-linux/blob/debian-guivm/appvm-scripts/etc/X11/Xsession.d/60xfce-desktop